plugins/ocm-backend/package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "build": "backstage-cli package build",
     "clean": "backstage-cli package clean",
-    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package --embed-package @backstage/plugin-kubernetes-common --no-embed-as-dependencies",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --no-embed-as-dependencies",
```

Comment:
The CI tests were failing because the `ocm-backend` was embedding `@backstage/plugin-kubernetes-common` in export-dynamic script whereas other backend plugin does not embed it. The embedded package is internally pinned to a older version of `kubernetes/client-node` causing the conflict with the PR changes.

https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/package.json#L34

Reply:
@PatAKnight are you ok with this change?

Reply:
It's probably being embedded because the ocm plugin has a runtime dependency on this package, and it won't be present in the package search path when run as a dynamic plugin.  I recall there was a general issue to remove the dependency on the kubernetes plugin code from a few plugins, so if this has been changed as well then this is probably going to be fine.  Otherwise, if this code is still using the kubernetes plugin API client, this change will cause this plugin to break when run in RHDH.

Reply:
Took a slightly closer look at the OCM code in particular, I don't see it actually using this dependency at a quick glance and now I'm wondering why it's even listed in the [package.json](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/package.json#L56) for this plugin.  It's worth trying to clean this up if possible.

Reply:
@gashcrumb Thanks for the review. I see there is a const being imported from this pacakge in [ManagedClusterProvider.ts](https://github.com/janus-idp/backstage-plugins/blob/218c4f0776cad63f62ade6489a5d612e4ab444a8/plugins/ocm-backend/src/providers/ManagedClusterProvider.ts#L33). I have now defined that const locally in `constants.ts` and removed the dependancy of `@backstage/plugin-kubernetes-common` from package.json.